.forge/skills/aspire/SKILL.md

```
@@ -0,0 +1,172 @@
+    .WithReference(orderApi); // paymentApi can discover orderApi
+
+// In PaymentApi — use service discovery
+builder.Services.AddHttpClient<OrderClient>(client =>
+{
+    client.BaseAddress = new Uri("https+http://order-api");
+});
+```
+
+### Solution Structure with Aspire
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)